src/map/pc.cpp

```diff
@@ -6259,6 +6259,18 @@ bool pc_isUseitem(map_session_data *sd,int n)
 	if( itemdb_group.item_exists(IG_MERCENARY, nameid) && sd->md != nullptr )
 		return false; // Mercenary Scrolls
 
+	// Safe check type cash disappear when overweight [Napster]
+	if( item->type == IT_CASH ){
+		if( pc_is90overweight(sd) ) {
+			clif_msg(sd, MSI_CANT_GET_ITEM_BECAUSE_WEIGHT);
+			return false;
+		}
+		if( !pc_inventoryblank(sd) ) {
+			clif_messagecolor(&sd->bl, color_table[COLOR_RED], msg_txt(sd, 732), false, SELF); //Item cannot be open when inventory is full
+			return false;
+		}
+	}
```

Comment:
We can't just revert this, because should be the same as the checks for itemgroup items and there's no way to check for the amount here. I think we should add checks for getitem and rentitem instead